### PR TITLE
Small pillow tweaks

### PR DESCRIPTION
**Pull request by czue**
goes with https://github.com/dimagi/pillowtop/pull/176

@snopoke @calellowitz :blowfish:

### CODE REVIEW

File: corehq/pillows/cacheinvalidate.py [hunk 1]
```diff
@@ -16,10 +17,8 @@ class CacheInvalidatePillow(BasicPillow):
     couch_filter = "hqadmin/not_case_form"  # string for filter if needed
 
 
-
-    def __init__(self, **kwargs):
-        super(CacheInvalidatePillow, self).__init__(**kwargs)
-        self.couch_db = Domain.get_db()
+    def __init__(self):
```
Comment (dimagimon) [inline corehq/pillows/cacheinvalidate.py]: E303 too many blank lines (2)

File: testapps/test_pillowtop/tests/test_settings.py [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 from django.conf import settings
-from django.test import TestCase
+from django.test import TestCase, override_settings
 import json
 from corehq.util.test_utils import TestFileMixin
 from pillowtop.listener import AliasedElasticPillow, BasicPillow
 from pillowtop.utils import get_all_pillow_configs
 
 
+@override_settings(DEBUG=True)
```
Comment (snopoke) [inline testapps/test_pillowtop/tests/test_settings.py]: Why did you need this?

Comment (snopoke) [inline testapps/test_pillowtop/tests/test_settings.py]: To make the asserts fail?

Comment (czue) [inline testapps/test_pillowtop/tests/test_settings.py]: yeah, i ran into [this](https://docs.djangoproject.com/en/1.4/topics/testing/#other-test-conditions) trying to make a test fail earlier

Comment (snopoke) [inline testapps/test_pillowtop/tests/test_settings.py]: Yea, just wondering if there should be an option in soft assert to also
fail in unit tests.

On 10 November 2015 at 20:56, Cory Zue <notifications@github.com> wrote:

> In testapps/test_pillowtop/tests/test_settings.py
> <https://github.com/dimagi/commcare-hq/pull/9046#discussion_r44448037>:
>
> >  import json
> >  from corehq.util.test_utils import TestFileMixin
> >  from pillowtop.listener import AliasedElasticPillow, BasicPillow
> >  from pillowtop.utils import get_all_pillow_configs
> >
> >
> > +@override_settings(DEBUG=True)
>
> yeah, i ran into this
> <https://docs.djangoproject.com/en/1.4/topics/testing/#other-test-conditions>
> trying to make a test fail earlier
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9046/files#r44448037>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa

Comment (czue) [inline testapps/test_pillowtop/tests/test_settings.py]: yeah, i actually added it at first yesterday and decided this was simpler. happy to add it back if you think that's better

Comment (snopoke) [inline testapps/test_pillowtop/tests/test_settings.py]: Seems fine for just this case but if we start using it like this more then
we should switch to the other approach.

On 11 November 2015 at 09:50, Cory Zue <notifications@github.com> wrote:

> In testapps/test_pillowtop/tests/test_settings.py
> <https://github.com/dimagi/commcare-hq/pull/9046#discussion_r44508545>:
>
> >  import json
> >  from corehq.util.test_utils import TestFileMixin
> >  from pillowtop.listener import AliasedElasticPillow, BasicPillow
> >  from pillowtop.utils import get_all_pillow_configs
> >
> >
> > +@override_settings(DEBUG=True)
>
> yeah, i actually added it at first yesterday and decided this was simpler.
> happy to add it back if you think that's better
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9046/files#r44508545>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa
